7weeks/src/main/java/Formula.java

```diff
@@ -0,0 +1,43 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class Formula {
+    private static String[] arrayFormula;
+
+    public void inputFormula() throws IOException {
+        BufferedReader bufferedReader = new BufferedReader((new InputStreamReader(System.in)));
+        this.arrayFormula = bufferedReader.readLine().split(" ");
+    }
+
+    public void startCheckOperand() throws IllegalArgumentException {
+        boolean isComputable = true;
+        int checkIndex = 1;
+        for (checkIndex = 1; checkIndex < arrayFormula.length + 1; checkIndex += 2) {
+            isComputable = isOperandComputable(checkIndex);
+            if (isComputable == false){
```

Comment:
`isCoumputable`변수를 사용하지 않고
```suggestion
            if (isOperandComputable(checkIndex)){
```
로 할 수 있지않을까요? 대신 `return`결과나 내부 로직, 메서드는 조금 수정이 필요할거에요